WebHostLib/downloads.py

```diff
@@ -83,6 +83,8 @@ def download_slot_file(room_id, player_id: int):
             fname = f"AP_{app.jinja_env.filters['suuid'](room_id)}_SP.apsm64ex"
         elif slot_data.game == "Dark Souls III":
             fname = f"AP_{app.jinja_env.filters['suuid'](room_id)}.json"
+        elif slot_data.game == "Overcooked! 2":
+            fname = f"OC2Modding.json"
```

Comment:
It's missing from the setup guide but like sm64 and v6 the json is only created for solo seeds for offline play. Curious how this works with co-op as I'm not sure that was tested, but probably just requires everyone get the json file. I know slot_data is being used normally though because I wrote it ;P

Reply:
I think I will just remove the download. Less things to support, and using the slot is the best way to play imo.